pyspecdata/core.py

```diff
@@ -1333,14 +1334,16 @@ def show_array(x, indent=""):
             else:
                 return x
 
+        if self.data.size < 2:
+            retval = str(self.data.item())
+            if self.get_error() is not None:
+                retval += "±" + str(self.get_error().item())
+            return retval
```

Comment:
It think address this seperately -- this is not about 1D

Reply:
if the self is a single number (e.g. the mean) then there is no need to show the array or dimlabels and axes etc. of the nddata since it's just a number so it just adds the error to the single datapoint and calls it a day for the return